$DragonFly: src/secure/usr.bin/ssh/Attic/ssh.1.no_obj.patch,v 1.3 2006/02/13 14:47:58 corecode Exp $
--- ssh.1	2006-01-31 11:47:59.000000000 +0100
+++ ssh.1	2006-02-12 03:05:52.000000000 +0100
@@ -678,7 +678,7 @@
 If the machine the user logs in from is listed in
 .Pa /etc/hosts.equiv
 or
-.Pa /etc/shosts.equiv
+.Pa /etc/ssh/shosts.equiv
 on the remote machine, and the user names are
 the same on both sides, or if the files
 .Pa ~/.rhosts
@@ -964,6 +964,7 @@
 .Ev DISPLAY .
 Forwarding of X11 connections can be
 configured on the command line or in configuration files.
+Take note that X11 forwarding can represent a security hazard.
 .Pp
 The
 .Ev DISPLAY
@@ -1277,7 +1278,7 @@
 This file is for host-based authentication (see above).
 It should only be writable by root.
 .Pp
-.It /etc/shosts.equiv
+.It /etc/ssh/shosts.equiv
 This file is used in exactly the same way as
 .Pa hosts.equiv ,
 but allows host-based authentication without permitting login with
